dask/dataframe/tests/test_shuffle.py

```diff
@@ -640,13 +640,7 @@ def test_set_index_interpolate(engine):
 
     d1 = d.set_index("x", npartitions=3)
     assert d1.npartitions == 3
-    if engine == "cudf":
-        # cuDF has to use "linear" interpolation as "nearest" interpolation
-        # is not supported by CuPy. The result for d1.divisions is different
-        # in this case.
-        assert set(d1.divisions) == set([1, 2, 3, 4])
-    else:
-        assert set(d1.divisions) == set([1, 2, 4])
+    assert set(d1.divisions) == set([1, 2, 4])
```

Comment:
cc @jrbourbeau ( as this relates to issue https://github.com/dask/dask/issues/8063 )

Reply:
Thanks for the quick fix @galipremsagar. Just for my own understanding, we dropped using "nearest" interpolation for cuDF over in https://github.com/dask/dask/pull/8029, so now we always use the same type of interpolation, regardless of the DataFrame parition type (e.g. pandas vs. cuDF)?

Reply:
Yes, since `nearest` is now supported from cupy side.